<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailLoginWidget</class>
 <widget class="QWidget" name="EmailLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999999</width>
    <height>999999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Google Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="mainLayout">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Outer container */
QWidget#mainLayout {
    border-radius: 8px;        /* slightly smoother corners */
    border: 3px solid #000000; /* black border */
    padding: 15px;             /* space inside the container */
    background-color: #fdfdfd; /* subtle neutral background */
}

/* Labels inside the container */
QLabel {
    font-size: 14px;
    padding-bottom: 4px;
}

/* Line edits */
QLineEdit {
    border-radius: 5px;
    border: 1px solid #b0b0b0;
    padding: 5px 8px;
    min-height: 28px;
}

/* Checkbox */
QCheckBox {
    spacing: 4px;
    padding-top: 4px;
}

/* Buttons */
QPushButton {
    border-radius: 6px;         
    border: 1px solid #808080;
    padding: 6px 12px;   
    max-width: 60em;        
    background-color: #ffffff;  /* subtle neutral background for button */
}

QPushButton:hover {
    border-width: 2px;
    border-color: #606060;
}

QPushButton:pressed {
    border-style: inset;      
}
</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>25</number>
          </property>
          <property name="rightMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>Email Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="emailLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>25</number>
          </property>
          <property name="rightMargin">
           <number>25</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="passLabel">
            <property name="text">
             <string>App Password: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="appPasswordEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="passCheckBox">
            <property name="text">
             <string>Show Pass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>986</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
